docs/notebooks/configuration.ipynb

```diff
@@ -46,11 +46,11 @@
       "    optim: Adam\n",
       "    optim_args:\n",
       "      lr: 0.001\n",
-      "    loss_fn_path: ./loss/celoss.py\n",
+      "    loss_fn_path: ./resources/loss/celoss.py\n",
```

Comment:
I think depending on the current working directory, this relative path will be correct if the user is in the examples folder. If the user is in the main APPFL or any subdirectory (e.g., globus_compute compute for to do `python run.py` as given in the tutorials from appfl.ai, this relative path will be incorrect. This is the same for the client configuration in globus compute case (under config_gc). I am not sure if we should change these paths or maybe leave a note to users in the tutorials.

Reply:
This change is due to a previous resource refactor which puts `loss` folder into `resources/loss` folder. However, we did not re-run this notebook since then, so the path is not changed until now.